features/admin/deactivate_supplier_user.feature

```diff
@@ -28,14 +30,31 @@ Scenario Outline: Correct users can deactivate and reactivate a supplier's contr
     | role  |
     | admin |
 
-Scenario Outline: Correct users cannot update suppliers users
+Scenario Outline: Correct users can view suppliers users
+  Given I am logged in as the production <role> user
+  And I click the '<link-name>' link
+  And I enter 'DM Functional Test Supplier' in the 'Find a supplier by name' field
+  And I click the 'find_supplier_by_name_search' button
+  When I click the summary table 'Users' link for 'DM Functional Test Supplier'
+  Then I don't see the 'Deactivate' button
+  And I see an entry in the 'Users' table with:
+    | Name                               | Email address                                                    | Last login  | Pwd changed | Locked | Status |
+    | DM Functional Test Supplier User 1 | dm-functional-test-supplier-user-1@digital.cabinet-office.gov.uk | <ANY>       | <ANY>       | No     | Active |
+  And I see an entry in the 'Users' table with:
+    | Name                               | Email address                                                    | Last login  | Pwd changed | Locked | Status |
+    | DM Functional Test Supplier User 2 | dm-functional-test-supplier-user-2@digital.cabinet-office.gov.uk | <ANY>       | <ANY>       | No     | Active |
+
+  Examples:
+    | role                    | link-name                   |
+    | admin-ccs-category      | Edit suppliers and services |
+    | admin-framework-manager | View suppliers and services |
+
+Scenario Outline: Correct users cannot view suppliers users
```

Comment:
I know it's not your naming, but "correct...cannot" is a very confusing way of explaining what's being tested here... I think I'd maybe write "users with insufficiently privileged roles cannot" or something.

Reply:
https://github.com/alphagov/digitalmarketplace-functional-tests/pull/428#issuecomment-355919388

Reply:
Addressed elsewhere. Going to do it in another PR. This PR i to get the functional tests green again